test/Datadog.Trace.Tests/PlatformHelpers/ContainerMetadataTests.cs

```diff
@@ -60,14 +60,29 @@ public class ContainerMetadataTests
 3:blkio:/ecs/55091c13-b8cf-4801-b527-f4601742204d/432624d2150b349fe35ba397284dea788c2bf66b885d14dfc1569b01890ca7da
 2:memory:/ecs/55091c13-b8cf-4801-b527-f4601742204d/432624d2150b349fe35ba397284dea788c2bf66b885d14dfc1569b01890ca7da
 1:name=systemd:/ecs/55091c13-b8cf-4801-b527-f4601742204d/432624d2150b349fe35ba397284dea788c2bf66b885d14dfc1569b01890ca7da
+";
+
+        public const string Fargate1Dot4 = @"
```

Comment:
Stylecop wouldn't let me use an underscore in the name, which was my preference

![Your move, creep](https://media.giphy.com/media/1KYu5qP0chFPa/giphy.gif)